src/pages/git/git-hosting/index.md

```diff
@@ -3,7 +3,7 @@ title: Git hosting
 ---
 ## Git hosting
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/tagging-in-git/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This is a stub. <a href='https://github.com/freeCodeCamp/guides/blob/master/src/pages/git/git-hosting/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```

Comment:
These stubs are usually generated by the `bin/normaliseArticles.js` script. It uses the following template: `https://github.com/freecodecamp/guides/tree/master/src/pages/${filePath}/index.md`. The correct link here would be `https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-hosting/index.md`. Any other link would be changed the next time that script runs.

Reply:
Yes, still the wrong link... How should a stub be created then if there's a script?

Reply:
@thomaskuntzz `bin/normaliseArticles.js` is the script that creates stubs, but it also does some other things that could mess with the repo right now 😅 We're working on cleaning it up (#7825 #752)